corehq/apps/dump_reload/tests/test_couch_dump_load.py

```diff
@@ -100,6 +101,11 @@ def _dump_line_to_doc_class(line):
     def test_docs_with_domain(self):
         # one test for all docs that have a 'domain' property
         doc_types = []
+        special_handlers = {
+            "DataSourceConfiguration": {
+                "referenced_doc_type": lambda x: random.choice(VALID_REFERENCED_DOC_TYPES)
```

Comment:
Not a huge fan of randomness in tests like this, but did it to keep it in line with the randomness in the original test